### DISCUSSION TITLE
How do I disable documentation from single line comments?

### DISCUSSION THREAD

**Discussion by BlueStaggo** [microsoft/vscode-cpptools#7902] [Q&A]
This is annoying:
![image](https://user-images.githubusercontent.com/71090710/127745974-c4df792d-9f96-4632-bf5f-7ed72935a2d5.png)
The comment is not intending to describe the variable; it is supposed to show what step is being done. How do I disable this and only let doc comments (`/** Like this */`) add information on hovering over a variable? If this is impossible, I'll post an issue about this.

**Comment by sean-mcmanus** [comment]
I don't think there's a way to disable it. Can you file a feature request?
